### DISCUSSION TITLE
[RELEASE] Farmhouse (from SAS Zombie Assault 3)

### DISCUSSION THREAD

**Discussion by Revnova** [nzp-team/nzportable#782] [Map Releases]
# This is a migration of slugparty's map release ``farmhouse``. Below is mostly the original post. You can find the original post [here](https://nzportable.forumotion.com/t2154-release-farmhouse-from-sas-zombie-assault-3)

![500px-Farmhouse](https://github.com/nzp-team/nzportable/assets/34180180/91dc2aa4-6fe5-4f5c-950a-b02ce5130b87)

YOU ALREADY KNOW WHAT IT IS
(updated 6/21/2016)
# [Download Here](https://github.com/user-attachments/files/15858920/Farmhouse.zip)

Thanks so much to Douch_Dagger for the suggestion!

I hope to update this map significantly in the future so as always I am open to any suggestions or constructive criticisms.

![28176e10](https://github.com/nzp-team/nzportable/assets/34180180/5028bcc7-2b49-4f47-8870-07fc02b83a40)
![bf596b10](https://github.com/nzp-team/nzportable/assets/34180180/21bda244-0f82-407e-b9a0-8919f3d85f9a)

EDIT: Updated, added lots of wall guns

**Comment by KrEd1bLe** [comment]
awesome map! simple but sweet :)

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org
